README.md

```diff
@@ -1,4 +1,36 @@
 <p align="center"><img src="https://avatars1.githubusercontent.com/u/58499608?s=100&v=4" alt="One Platform Logo"></p>
 
 # One Platform
-An integrated application hosting platform.
+An integrated application hosting platform which allows you to host your SPAs. It also provides common services like
+- Authentication and Authorization
+- Notifications Framework
+- Feedback Framework
+
+![Version](https://img.shields.io/badge/version-0.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license/1-Platform/one-platform)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
```

Comment:
Can you move these after line 3.

The shields should be below the project title (i.e. the heading)

Reply:
![Screenshot from 2020-02-28 12-02-15](https://user-images.githubusercontent.com/12994292/75516333-3a328a00-5a22-11ea-9dc1-f4026f8086bb.png)
Done